modules/ROOT/attachments/cloud-controlplane-api.yaml

```diff
@@ -3157,7 +3182,7 @@ paths:
                     For example, if the name of a tag is "gcp.network-tag.network-tag-foo", the network tag named
                     "network-tag-foo" will be added to the Redpanda cluster GKE nodes.
                     Note: The value of a network tag will be ignored.
-                    See the official [GCP VPC](https://cloud.google.com/vpc/docs/add-remove-network-tags) for more details on network tags.
+                    See the details on network tags at https://cloud.google.com/vpc/docs/add-remove-network-tags.
```

Comment:
See the official [GCP VPC](https://cloud.google.com/vpc/docs/add-remove-network-tags) for more details on network tags.